--- pango-1.36.8/pango/pango-layout.c~	2014-08-30 18:11:04.000000000 +0000
+++ pango-1.36.8/pango/pango-layout.c	2016-04-24 02:07:43.016315933 +0000
@@ -3382,7 +3382,7 @@
  * or %BREAK_NONE_FIT returned. This is used when the end of the
  * run is not a break position.
  */
-static BreakResult
+__attribute__((target_clones("arch=core-avx2","default"))) static BreakResult
 process_item (PangoLayout     *layout,
 	      PangoLayoutLine *line,
 	      ParaBreakState  *state,
